setup.py

```diff
@@ -37,6 +37,16 @@
     packages=['dukpy', 'dukpy.webassets'],
     ext_modules=[duktape],
     install_requires=INSTALL_REQUIRES,
+    extras_require={
+        'testing': [
+            'coveralls',
+            'nose',
+            'mock',
+        ],
+        'webassets': [
```

Comment:
This is just for convenience: it's not a mandatory requirement, but if users want extra features they would be able to be verbose depending on it. I usually treat this as a public API.